### PR TITLE
Tasks: Track bridger tasks base config

### PR DESCRIPTION
**Pull request by PatricioHenderson**

### CODE REVIEW

File: src/Task.ts [hunk 1]
```diff
@@ -64,6 +68,19 @@ export function handleCustomMaxSlippageSet(event: CustomMaxSlippageSet): void {
...
+export function handleCustomDestinationChainSet(event: CustomDestinationChainSet): void {
+  const task = Task.load(event.address.toHexString())
+  if (task == null) return log.warning('Missing task entity {}', [event.address.toHexString()])
+
+  const customDestinationChainId = getTaskCustomConfigId(task, event.params.token)
+  let customDestinationChain = CustomDestinationChain.load(customDestinationChainId)
+  if (customDestinationChain === null) customDestinationChain = new CustomDestinationChain(customDestinationChainId)
+  customDestinationChain.task = task.id
+  customDestinationChain.token = loadOrCreateERC20(event.params.token).id
+  customDestinationChain.destinationChain = event.params.defaultDestinationChain
```
Comment (facuspagnuolo) [inline src/Task.ts:80]: Should be `event.params.destinationChain`, do you mind fixing this in the contract?

File: abis/IAllTasks.json [hunk 2]
```diff
@@ -1606,5 +1606,50 @@
...
+      {
+        "indexed": true,
+        "internalType": "address",
+        "name": "token",
+        "type": "address"
+      },
+      {
+        "indexed": true,
+        "internalType": "uint256",
+        "name": "defaultDestinationChain",
```
Comment (facuspagnuolo) [inline abis/IAllTasks.json:1648]:
```suggestion
        "name": "destinationChain",
```

File: schema.graphql [hunk 3]
```diff
@@ -95,31 +95,48 @@ type Task @entity {
...
 }
 
-type AcceptanceList @entity {
+type CustomMaxSlippage @entity {
   id: ID!
   task: Task!
-  type: String!
-  tokens: [String!]!
+  token: ERC20!
+  slippage: BigInt!
```
Comment (facuspagnuolo) [inline schema.graphql:132]: This one was carried from a previous PR, do you mind renaming it to `maxSlippage`?

**Review by facuspagnuolo — APPROVED** [review]
Looks great!

File: src/Task.ts (hunk 1 above)
Comment (PatricioHenderson) [reply src/Task.ts:80]: My pleasure !  https://github.com/mimic-fi/v3-core/pull/102

File: schema.graphql (hunk 3 above)
Comment (PatricioHenderson) [reply schema.graphql]: Sure, no problem

**Pull request merged**